src/dbcp/data_mart/projects.py

```diff
@@ -15,20 +15,106 @@
 from dbcp.helpers import get_sql_engine
 
 
-def _get_and_join_iso_tables(engine: sa.engine.Engine) -> pd.DataFrame:
+def _merge_with_gridstatus(
+    lbnl: pd.DataFrame, engine: sa.engine.Engine
+) -> pd.DataFrame:
+    """Merge non ISO LBNL projects with ISO projects in GridStatus.
+
+    Args:
+        lbnl: lbnl ISO queue projects
+        engine: engine to connect to the local postgres data warehouse
+    """
+    is_iso = ~lbnl.iso_region.str.contains("non-ISO")
+    lbnl_non_isos = lbnl[~is_iso].copy()
+
+    with engine.connect() as con:
+        query = """
+            WITH
+            gs AS (
+                SELECT
+                    county_id_fips,
+                    queue_id,
+                    is_nearly_certain,
+                    state_id_fips,
+                    project_id,
+                    project_name,
+                    capacity_mw,
+                    developer,
+                    entity,
+                    utility,
+                    proposed_completion_date AS date_proposed_online,
+                    point_of_interconnection,
+                    UPPER(region) AS iso_region,
+                    is_actionable,
+                    resource_clean,
+                    queue_status,
+                    queue_date AS date_entered_queue,
+                    interconnection_status_raw AS interconnection_status
```

Comment:
I'm using `interconnection_status_raw` for now because we map the raw data directly to nearly certain and actionable statuses.